packages/Navbar/src/index.js

```diff
@@ -0,0 +1,29 @@
+import Inferno   from 'inferno'
+import Component from 'inferno-component'
+import { bind }  from 'decko'
+
+export class Navbar extends Component {
+
+  @bind
+  getStyles() {
+    const { backgroundColor, color } = this.props
```

Comment:
Use just background, not backgroundColor, for both consistency, simplicity and convention